Remove SP2019 specifics and Add multiple Motor Boards (#87)
sourcebots/sbot: docs/source/api/motor-board.rst

Accessing the Motor Board
-------------------------

If there is exactly one motor board attached to your robot, it can be accessed 
using the ``motor_board`` property of the ``Robot`` object.

.. code:: python

   my_motor_board = r.motor_board

.. Warning:: If there is more than one motor board on your kit, you *must* use the
  ``motor_boards`` property. ``r.motor_board`` *will cause an error*. This is because 
  the kit doesn't know which motor board you want to access.

> Motor boards attached to your robot  can be accessed under the ``motor_boards`` 

Reviewer: ```suggestion
Motor boards attached to your robot can be accessed under the ``motor_boards`` 
```

> property of the ``Robot``. The boards are indexed by serial number, will be written on the board.

Reviewer: ```suggestion
property of the ``Robot``. The boards are indexed by their serial number, which is written on the board.
```
